### ISSUE TITLE
Some of the operators in the model are not supported by the standard TensorFlow Lite runtime.

### ISSUE THREAD

**Issue by devbk007** [tensorflow/tensorflow#36015]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10
- TensorFlow installed from (source or binary):
- TensorFlow version (or github SHA if from source):1.15.0

**Provide the text output from tflite_convert**
Some of the operators in the model are not supported by the standard TensorFlow Lite runtime. If those are native TensorFlow operators, you might be able to use the extended runtime by passing --enable_select_tf_ops, or by setting target_ops=TFLITE_BUILTINS,SELECT_TF_OPS when calling tf.lite.TFLiteConverter(). Otherwise, if you have a custom implementation for them you can disable this error with --allow_custom_ops, or by setting allow_custom_ops=True when calling tf.lite.TFLiteConverter(). Here is a list of builtin operators you are using: AVERAGE_POOL_2D, CAST, CONCATENATION, CONV_2D, EXPAND_DIMS, FULLY_CONNECTED, MAX_POOL_2D, RESIZE_BILINEAR, SOFTMAX. Here is a list of operators for which you will need custom implementations: DecodeJpeg.
```
# Copy and paste here
```

Also, please include a link to a GraphDef or the model if possible.

**Any other info / logs**

Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by gadagashwini** [comment]
@devbk007, can you provide the complete code to analyze the issue. Thanks

**Comment by gadagashwini** [comment]
@devbk007, Provide the minimal code snippet to replicate the issue. Thanks!

**Comment by gadagashwini** [comment]
@devbk007, Please take a look at the [#35449](https://github.com/tensorflow/tensorflow/issues/35449#issuecomment-570318723) comment to solve the reported issue. Thanks!

**Comment by gadagashwini** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
